reactivesocket-cpp/src/ReactiveStreamsCompat.h

```diff
@@ -2,7 +2,7 @@
 
 #pragma once
 
-#include "reactive-streams-cpp/ReactiveStreams.h"
+#include <ReactiveStreams.h>
```

Comment:
Sure one can always create a collision.

I hope you agree that when someone decides to create a fork, it would be much if one only had to change the project name instead of renaming all public headers (well, only one, for now).

If your point is that there is no rock-solid way of generating unique names without a single source of truth, then I do agree. There are heuristics that many C++ projects try to follow to make collisions less likely. `<project name>/<header file>` is the one I've seen most often. If project names of two different projects are the same then ¯\_(ツ)_/¯.